src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return function(x) {
+    return function(y) {
+      const sum = x + y;
```

Comment:
Move the 'const sum' declaration to after the if statement, so it is only calculated when needed.
You can immediately return the amount without adding a variable here.